mlflow/pyfunc/backend.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         self,
         config,
         workers=1,
-        env_manager=_EnvManager.VIRTUALENV,
+        env_manager=None,
```

Comment:
We should also update the default value for `env_manager` argument in CLI/Python API to None. Otherwise the default experience creates unnecessary virtualization.